lib/cocoapods/installer/xcode/pods_project_generator/app_host_installer.rb

```diff
@@ -21,47 +21,53 @@ class AppHostInstaller
           #
           attr_reader :platform
 
-          # @return [String] the name to use for this app host target.
+          # @return [String] the name of the subspec.
           #
-          attr_reader :name
+          attr_reader :subspec_name
 
-          # @return [String] the name of the pod the app host installer will be installing within.
+          # @return [String] the name of the spec the app host installer will be installing within.
           #
-          attr_reader :pod_name
+          attr_reader :spec_name
+
+          # @return [String] the name of the app target label that will be used.
+          #
+          attr_reader :app_target_label
 
           # Initialize a new instance
           #
           # @param [Sandbox] sandbox @see #sandbox
           # @param [Pod::Project] project @see #project
           # @param [Platform] platform @see #platform
-          # @param [String] name @see #name
-          # @param [String] pod_name @see #pod_name
+          # @param [String] subspec_name @see #subspec_name
+          # @param [String] spec_name @see #spec_name
+          # @param [String] app_target_label see #app_target_label
           #
-          def initialize(sandbox, project, platform, name, pod_name)
+          def initialize(sandbox, project, platform, subspec_name, spec_name, app_target_label)
```

Comment:
`subspec_name` is awfully specific for the needs of a label here. Is there a better name for this? I do not quite understand the difference of passing a `spec_name` and `subspec_name` easily.